src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private String color;
+
+    private int number;
```

Comment:
```suggestion
    private String color;
    private int number;
```
pay attention for empty lines